src/components/Datasets.js

```diff
@@ -0,0 +1,232 @@
+
+import theme from '../theme';
+import Main from './common/Main';
+
+const fakeDatasets = [
+  {
+    name: 'Atomic Structure',
+    description: 'Atomic Structure Dataset of students during two months',
+    size: 413,
+    created: '7/13/20',
+    lastModified: '7/13/20',
+    anonymized: false,
+  },
+  {
+    name: 'Animals of the world',
+    description: 'Collection of all the referenced animals of world.',
+    size: 409600,
+    created: '7/12/18',
+    lastModified: '9/14/20',
+    anonymized: false,
+  },
+  {
+    name: 'Anonymized Atomic Structure',
+    description: 'Anonymized version of Atomic Structure Dataset',
+    size: 211,
+    created: '7/02/20',
+    lastModified: '7/02/20',
+    anonymized: true,
+  },
+];
+
+function descendingComparator(a, b, orderBy) {
+  if (b[orderBy] < a[orderBy]) {
+    return -1;
+  }
+  if (b[orderBy] > a[orderBy]) {
+    return 1;
+  }
+  return 0;
+}
+
+function getComparator(isAsc, orderBy) {
+  return isAsc
+    ? (a, b) => -descendingComparator(a, b, orderBy)
+    : (a, b) => descendingComparator(a, b, orderBy);
+}
+
+function stableSort(array, comparator) {
+  const stabilizedThis = array.map((el, index) => [el, index]);
+  stabilizedThis.sort((a, b) => {
+    const order = comparator(a[0], b[0]);
+    if (order !== 0) return order;
+    return a[1] - b[1];
+  });
+  return stabilizedThis.map((el) => el[0]);
+}
+
+const useStyles = makeStyles({
+  addButton: {
+    color: theme.palette.secondary.main,
+    backgroundColor: theme.palette.primary.main,
+    position: 'absolute',
+    right: '16px',
+    bottom: '16px',
+    '&:hover, &.Mui-focusVisible': {
+      backgroundColor: darken(theme.palette.primary.main, 0.2),
+    },
+  },
+});
+
+const Datasets = () => {
+  const classes = useStyles();
+
+  // name column sorted ascending by default
+  const [isAsc, setIsAsc] = React.useState(true);
+  const [orderBy, setOrderBy] = React.useState('name');
+
+  const handlePublish = () => {
+    // TODO: implement publish functionality
+  };
+
+  const handleAnonymize = () => {
+    // TODO: implement anonymize functionality
+  };
+
+  const handleEdit = () => {
+    // TODO: implement edit functionality
+  };
+
+  const handleDelete = () => {
+    // TODO: implement delete functionality
+  };
+
+  const handleAdd = () => {
+    // TODO: implement add functionality
+  };
+
+  const handleSortColumn = (column) => {
+    if (orderBy === column) {
+      setIsAsc(!isAsc);
+    } else {
+      setOrderBy(column);
+      setIsAsc(true);
+    }
+  };
+
+  return (
+    <Main content>
+      <Container>
+        <h1>Datasets</h1>
+        <Table aria-label="simple table">
+          <TableHead>
+            <TableRow>
+              <TableCell align="left">
+                <TableSortLabel
+                  active={orderBy === 'name'}
+                  direction={orderBy === 'name' && !isAsc ? 'desc' : 'asc'}
+                  onClick={() => handleSortColumn('name')}
```

Comment:
Let's refactor the column names as constants and saved in `src/config/constants`. Maybe have something similar to 
```
DATASETS_TABLE_COLUMNS = {
  NAME: 'name',
  SIZE: 'size'
}
```
etc..
You can check this example here https://github.com/graasp/graasp-desktop/blob/7cb48aa56424ed8b940770c7ca2c68532ebcb903/src/components/classrooms/StudentsTable.js . You might not need to make everything dynamic, but let's avoid writing repetitive strings.